### ISSUE TITLE
Message.Type.Recalled not catching recalled message any more

### ISSUE THREAD

**Issue by suntong** [wechaty/wechaty-puppet-padplus#263]
> Important：Please file the issue follow the template, or we won't help you to solve the problem.

## 0. Report Issue Guide

1. Please run the following command and check whether the problem has been fixed：

```
npm install wechaty-puppet-padlus@next
```

2. Please search in the issue first, and make sure your problem had not been reported before

## 1. Versions

- What is your wechaty, wechaty-puppet-padplus version?
Answer:

```
wechaty@0.40.10
wechaty-puppet-padplus@0.6.3 
```

- What is your node version? (run `node --version`)
Answer: v10.21.0

- What os are you using
Answer: Ubuntu 20.04 LTS

## 2. Describe the bug

My code handles `Message.Type.Recalled`, and it has been working fine, but lately it is broken.

## 3. To Reproduce

1. catch & handles `Message.Type.Recalled` in the code
2. send a recall message in wx
3. the `Message.Type.Recalled` is not caught

## 4. Expected behavior

the `Message.Type.Recalled` is caught & handled

## 5. Actual behavior

the `Message.Type.Recalled` is not caught & handled

## 6. Full Output Logs

The new recalled message is of format

```xml
<sysmsg type="revokemsg"><revokemsg><session>18323595546@chatroom</session><msgid>1692528799</msgid><newmsgid>5007192124798729796</newmsgid><replacemsg><![CDATA["someone" recalled a message]]></replacemsg></revokemsg></sysmsg>
```

I believe `Message.Type.Recalled` is handling the older format.

**Comment by su-chang** [comment]
Good catch!

It will be fixed in `wechaty-puppet-padplus@0.7.27`.

Thank you very much.

**Comment by suntong** [comment]
Fixed in latest padplus@0.7.27
